src/styles/BEM-blocks/card.scss

```diff
@@ -0,0 +1,140 @@
+@import "../utils/_variables.scss";
+
+.card {
+  /* display flex ect. */
+  box-sizing: border-box;
+  width: $card-bg-size-w;
+  height: $card-bg-size-h;
+  border-radius: $card-border-radius;
+  padding: 32px 16px 16px;
+
+  /* color design */
+  border: 1px solid $card-border-color;
+  background: $card-bg-light-color;
+  font-family: "Roboto", Helvetica, sans-serif;
+
+  /* animation -wyszczególnienie nie działało- */
+  transition-property: transform;
+  transition-duration: 300ms;
```

Comment:
that also can be comined into `transition`

and `animation` property is strictly related to `@key-frames`, transition is just a simple hack to delay some action so they look animated